app/config/custom-inflector-rules.js

```diff
@@ -1,43 +1,45 @@
 import Inflector from 'ember-inflector';
 
-const inflector = Inflector.inflector;
+export function registerInflectorRules() {
```

Comment:
In theory, this shouldn't be needed, since EmberData copies the already registered rules into its new system. However, due to a bug in ember-inflector itself that doesn't work as expected. This will most likely be fixed in a future release, but this unblocks us at least.